feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/home/HomeFragment.java

```diff
@@ -1,42 +1,256 @@
         homeViewModel = ViewModelProviders.of(this).get(HomeViewModel.class);
         View root = inflater.inflate(R.layout.fragment_home, container, false);
         homeViewModel.getText().observe(getViewLifecycleOwner(), new Observer<String>() {
             @Override
-            public void onChanged(@Nullable String s) {
-            }
-        });
+            public void onChanged(@Nullable String s) { }});
         return root;
     }
 
     @Override
-    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+    public void onResume() {
+        super.onResume();
+
+        UserDAO userDAO = new UserDAO();
+        userDAO.get(User.myTempUserName, new UserCallback() {
+            @Override
+            public void onCallback(User user) {
+                // current user the is object for the login user
+                currentUser = user;
+                myEmotionList.clear();
+                myCurrentMoodList = currentUser.getMoodHistory();
+                for(int i = 0; i < myCurrentMoodList.size(); i++)
+                {
+                    myEmotionList.add(myCurrentMoodList.get(i));
+                }
+                SwipeMenuListView = getView().findViewById(R.id.listView);
+                adapter = new EmotionBookAdapter(getContext(), R.layout.list_adapter_view, myEmotionList);
+                SwipeMenuListView.setAdapter(adapter);
+
+            }
+        }, new VoidCallback() {
+            @Override
+            public void onCallback() { }
+        });
+    } // end of onResume
+
+    @Override
+    public void onViewCreated(@NonNull final View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
+
+        UserDAO userDAO = new UserDAO();
+        userDAO.get(User.myTempUserName, new UserCallback() {
+            @Override
+            public void onCallback(User user) { //temporarily using User.myTempUserName until login stuff is done
+
+                currentUser = user;
+                myEmotionList.clear();
+                myCurrentMoodList = currentUser.getMoodHistory();
+                for(int i = 0; i < myCurrentMoodList.size(); i++) {
+                    myEmotionList.add(myCurrentMoodList.get(i));
+                }
+                SwipeMenuListView = view.findViewById(R.id.listView);
+                adapter = new EmotionBookAdapter(getContext(), R.layout.list_adapter_view, myEmotionList);
+                SwipeMenuListView.setAdapter(adapter);
+
+                SwipeMenuCreator creator = new SwipeMenuCreator() {
+
+                    @Override
+                    public void create(SwipeMenu menu) {
+                        // create "open" item
+                        SwipeMenuItem openItem = new SwipeMenuItem(
+                                getContext());
+                        // set item background
+                        openItem.setBackground(new ColorDrawable(Color.rgb(0xC9, 0xC9,

```

Comment:
is it possible to chain these methods so that it's more clear as to what properties are being set on what objects? 
ie. openItem.setWidth(170).setTitle("View").setTitleSize(18) (add appropriate tabbing)